resources/cluster-essentials/files/certificates.networking.internal.knative.dev.crd.yaml

```diff
@@ -1,31 +0,0 @@
-apiVersion: apiextensions.k8s.io/v1beta1
-kind: CustomResourceDefinition
-metadata:
-  annotations:
-    "helm.sh/resource-policy": keep
```

Comment:
`"helm.sh/resource-policy": keep` annotation tells Helm that resource should stay on cluster, but it's will unpinned from Helm release, so you should create upgrade job and remove whole crds related to `knative-serving`.

Reply:
we will create documentation how to cleanup the cluster after knative serving was removed. This guarantees that customers who used knative serving for their own functionality will not lose any configuration.

Reply:
Migration guide can be found here: https://github.com/kyma-project/kyma/pull/9487#pullrequestreview-493484269